### ISSUE TITLE
请问现在storage组件还不支持 scale操作或者hpa扩容吗？

### ISSUE THREAD

**Issue by kissingtiger** [vesoft-inc/nebula-operator#527]
请问现在storage组件还不支持 scale操作或者hpa扩容吗？

**Comment by kissingtiger** [comment]
Does the storage component currently not support scale operation or HPA expansion?

**Comment by wenhaocs** [comment]
No. Scaling storaged will cause data movement, which will affect your online query latency. Please let us know if you expect frequent scaling of storaged in your use case.
